xbmc/Util.h

```diff
@@ -204,6 +204,13 @@ class CUtil
    */
   static int GetRandomNumber();
 
+  static int ConvertSecsToOffset(double secs) { return static_cast<int>(secs * 75); }
+  static double ConvertOffsetToSecs(int offset) { return offset / 75.0; }
```

Comment:
why do we use this somehow strage offset units?

Reply:
> why do we use this somehow strage offset units?

this goes way back IMO. I think it's to have a precision of one frame, assuming 75fps. There are several places in the code that work with this. As I said, we can either clean this up a bit (like I'm proposing here, not too invasive) or go all the way and get rid of it completely (more risky).

Reply:
specifically it was the resolution of some timer in use on og xbox. i don't recall all the dirty details but that's the source of the weird number.

Reply:
I'd prefer the 2nd option. In current stage a little risk is acceptable. But it's your decision. Thanks very much for cleaning this up.

Reply:
don't know if I have the time for this - as I said this can go very deep and once beyond the simple conversion then the math with the frame counts kicks in and who knows how far that leads me. Probably not for inclusion in Leia v18.